<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ShareSettings</class>
 <widget class="QDialog" name="ShareSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Share settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Basic</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="folder" />
            </item>
            <item>
             <widget class="QPushButton" name="folderBrowse" >
              <property name="text" >
               <string>&amp;Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&amp;Port:</string>
            </property>
            <property name="buddy" >
             <cstring>port</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QComboBox" name="type" />
          </item>
          <item row="1" column="1" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QSpinBox" name="port" >
              <property name="maximum" >
               <number>65535</number>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="portPickRandom" >
              <property name="text" >
               <string>Pick &amp;random</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Folder:</string>
            </property>
            <property name="buddy" >
             <cstring>folder</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&amp;Type:</string>
            </property>
            <property name="buddy" >
             <cstring>type</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>&amp;Security</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Access permission</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="downloadOnly" >
            <property name="text" >
             <string>&amp;Download only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dropbox" >
            <property name="text" >
             <string>Drop&amp;box (allow upload but not modifying of existing files)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fullAccess" >
            <property name="text" >
             <string>&amp;Full access</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Authentication</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="anonymous" >
            <property name="text" >
             <string>Allow &amp;anonymous access</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="authentication" >
            <property name="text" >
             <string>Use password based &amp;authentication:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QLabel" name="usernameLabel" >
              <property name="text" >
               <string>&amp;Username:</string>
              </property>
              <property name="buddy" >
               <cstring>username</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="2" >
             <widget class="QLineEdit" name="password" >
              <property name="echoMode" >
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLabel" name="passwordLabel" >
              <property name="text" >
               <string>&amp;Password:</string>
              </property>
              <property name="buddy" >
               <cstring>password</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QLineEdit" name="username" />
            </item>
            <item row="0" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>folder</tabstop>
  <tabstop>folderBrowse</tabstop>
  <tabstop>port</tabstop>
  <tabstop>portPickRandom</tabstop>
  <tabstop>type</tabstop>
  <tabstop>downloadOnly</tabstop>
  <tabstop>dropbox</tabstop>
  <tabstop>fullAccess</tabstop>
  <tabstop>anonymous</tabstop>
  <tabstop>authentication</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="netek_icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ShareSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>400</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ShareSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>493</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
